src/style.css

```diff
@@ -0,0 +1,61 @@
+@font-face {
+  font-family: "Avenir";
+  src: url(fonts/Avenir-Book.ttf),
+    url(fonts/Avenir-Heavy.ttf);
+}
+
+body {
+  margin: 0;
+}
+
+.search-bar {
+  margin-top: 20px;
+}
+
+.search-bar__input {
+  width: 100%;
+  font-family: Avenir, sans-serif;
+  font-weight: 300;
+  font-style: normal;
+  line-height: normal;
+  border-radius: 4px;
+  border: 1px solid #e1e7ed;
+  flex-shrink: 0;
+  background-color: #fff;
+  box-shadow: 0 1px 8px 0 rgba(61, 78, 97, 0.1);
+}
+
+.search-bar__input--big {
+  padding: 23px 0 23px 61px;
+  font-size: 16px;
+  background: url("./images/Search.svg") 26px 24px no-repeat;
+  background-size: 19px;
+}
+
+.search-bar__input--small {
+  padding: 10px 0 10px 33px;
+  background: url("./images/Search.svg") 13px 15px no-repeat;
+  background-size: 11px;
+  font-size: 14px;
+}
+
+.search-bar__input::-webkit-input-placeholder {
+  color: #3d4e61;
+}
+
+.search-bar__input:hover {
+
```

Comment:
Remove empty line here

Reply:
I don't understand why tests didn't pass. I used "PerfectPixel' and ewerything matches but tests not passed.

Reply:
Not all is pixel perfect, double check the mockup